core/src/main/java/com/linecorp/armeria/client/HttpRequestSubscriber.java

```diff
@@ -334,6 +334,7 @@ private void failAndRespond(Throwable cause) {
         if (response.isOpen()) {
             response.close(cause);
             error = Http2Error.INTERNAL_ERROR;
+            // FIXME(ikhoon) fix here? wrap an exception with AbortedStreamException?
         } else if (cause instanceof WriteTimeoutException || cause instanceof AbortedStreamException) {
             error = Http2Error.CANCEL;
```

Comment:
Now, an abort can have various causes with `abort(Throwable)` and `abort(Supply<? extends Throwable>)`.

Reply:
I think it's fine to leave this as it is. If a user aborted with a different exception than `AbortedStreamException`, I'd say he or she wanted signal something else than an abort.